e2e/rbd.go

```diff
@@ -259,6 +259,15 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("create a PVC and validate owner", func() {
+				err := validateImageOwner(pvcPath, f)
```

Comment:
same tests need to be added for the rados namespace test also

Reply:
ok, will do!